### DISCUSSION TITLE
Contact MooseFS Team

### DISCUSSION THREAD

**Discussion by nxwrk** [moosefs/moosefs#541] [General]
Usually, I don't put these kind of trivial issues on Github, but our team is trying to reach out the MooseFS team in order to receive more information about MooseFS Pro.

Unfortunately, all our attempts via webpage contact form, email and phone remain unanswered.

**Comment by pkonopelko** [comment]
Hello @nxwrk, thank you for your message here on GitHub Discussions. We have checked this situation and unfortunately your requests were incorrectly filtered out as spam.

I have just sent you a reply via email and I truly apologise for the inconvenience this situation could have caused.
